src/Neo/SmartContract/ContractParameter.cs

```diff
@@ -182,6 +183,7 @@ private static JObject ToJson(ContractParameter parameter, HashSet<ContractParam
                             item["value"] = ToJson(p.Value, context);
                             return item;
                         }));
+                        context.Remove(parameter);
```

Comment:
It's a good fix, but how did you catch this issue? I'm asking because to me this bug seems a bit artificial, and it seems there's no way in real life to get such SC Param with cyclic reference.

Reply:
> It's a good fix, but how did you catch this issue? I'm asking because to me this bug seems a bit artificial, and it seems there's no way in real life to get such SC Param with cyclic reference from the node or dependent tools (except the tests, of course).

Code view.
Yes, there is currently no way to trigger this bug.